Remora.Sdk/README.md

```diff
@@ -47,12 +47,16 @@ The following properties are defined by the SDK.
 ### Framework Targets
 The following properties are defined by the SDK.
 
-| Property              | Value                        | Overridable |
-|-----------------------|------------------------------|-------------|
-| LibraryFrameworks*    | netstandard2.1;net6.0;net7.0 | Yes         |
-| ExecutableFrameworks* | net7.0                       | Yes         |
-| TargetFramework       |                              | Yes         |
-| TargetFrameworks      | (varies)                     | Yes         |
+| Property              | Value         | Overridable |
+|-----------------------|---------------|-------------|
+| LibraryFrameworks*    | net6.0;net7.0 | Yes         |
+| ExecutableFrameworks* | net7.0        | Yes         |
+| TargetFramework       |               | Yes         |
+| TargetFrameworks      | (varies)      | Yes         |
+| TargetNetStandard     | true          | Yes         |
+
+If `TargetNetStandard` is `true`, `netstandard2.1;` will be prepended to
+LibraryFrameworks: `netstandard2.1;net6.0;net7.0`.
```

Comment:
I wasn't quite sure how to properly document this. Do you think this works or should we do this differently?

Reply:
That works, though I might suggest something along these lines instead:

"If `TargetNetStandard` is `true`, `netstandard2.1` will be included as a target when building libraries. Set the property to `false` if this is undesirable."

The order (prepend, append, etc) is not really relevant for the end user in this case.